resources/assets/js/constants.js

```diff
@@ -11,6 +11,8 @@ export const MEMBER = 'Lid';
 export const OWNER = 'Eigenaar';
 export const ADMIN = 'Admin';
 
+export const API_PREFIX = '/api/v1';
```

Comment:
check with @delrueba  or this can be expanded to /api/v1/ui/ for all the javascript files
as the ui is the prefix of the user interface api

Reply:
Agreed.
If it's so that we'll only use /UI/ routes, this would indeed be better.